src/build.ts

```diff
@@ -137,12 +137,15 @@ interface BuildDockerImageParams {
 
 async function buildFileResource(plan: BuildPlan): Promise<void> {
   core.startGroup(`Build resource {plan.name}`)
-  await exec.exec(`./build-${plan.name}.sh`, [plan.source_file], {
```

Comment:
I think we can change this line to something like this: since `plan.source_file` equals `build-${plan.name}.sh`, we don't need to include it twice.

```
  await exec.exec(`./{plan.source_file}`, {
```

Reply:
i think i have done this...

the latest commit is:

```typescript
async function buildFileResource(plan: BuildPlan): Promise<void> {
  core.startGroup(`Build resource {plan.name}`)
  if (!plan.build_target) {
    throw new Error('build_target is required for file resources')
  }
  await exec.exec(`./${plan.source_file}`, [plan.build_target], {
    cwd: plan.source_directory
  })
   ...
```

Reply:
Oh, sorry I missed that. I actually think that since `plan.source_file` is `build-{resource_name}.sh`, which already includes the `plan.build_target` (also `resource_name`), the `plan.build_target` argument might be a bit redundant. However, I don't think this is critical, so either approach works for me—it's up to you. Once you've decided, you can change the action reference back to `canonical/operator-workflows/internal`, and I'll merge this pull request. I don't have any other comments, thanks!

Reply:
i think i'd like to leave it.  it helps the script know which target it's supposed to build.  If anything the build script can `> $1` or some non-sense to make sure the correct resource is built.